recipes-nodes/render-config/fetch-config.sh

```diff
@@ -11,8 +11,6 @@
 
 set -e
 
-source /etc/init-config.conf
```

Comment:
Don't need it since it only contained the address of BuilderHub

Reply:
Why are we hardcoding the address here and removing the ability to set this up during image building?
https://github.com/flashbots/meta-evm/blob/main/recipes-nodes/render-config/render-config.bb#L41

I remember we did this for the future to have the ability to set the INIT_CONFIG_URL during image build to other addresses in-case other operator would like to host their own secrets/configs

Reply:
`cvm-proxy-client` has the [hardcoded address](https://github.com/flashbots/meta-confidential-compute/blob/main/recipes-core/cvm-reverse-proxy-client/files/cvm-reverse-proxy-client-init) of the builder hub. Instead of doing `INIT_CONFIG_URL` here we should instead make it configurable in the `cvm-proxy-client` recipe, right?